### ISSUE TITLE
linking update for projects with DCP matrices

### ISSUE THREAD

**Issue by idazucchi** [ebi-ait/hca-ebi-wrangler-central#1226]
### Context
Updating the linking of a project generally requires the project to be soft deleted from Terra, however, it's unclear what effect the soft deletion would have on:
- DCP generated matrices
- contributor generated matrices added the MVP method for analysis files (see [specs](https://github.com/HumanCellAtlas/dcp2/blob/main/docs/dcp2_system_design.rst#511describing-cgms-as-supplementary-files)) 

### Description of the task:
We need to understand and then record :
- where the two type of files are stored - the same place as all the other prject files?
- if all the files are in the same area can some files be excluded from the soft deletion?

#### Example project - DCP generated matrices
short name: HumanAdultKidneyLiaoMo 
uuid: `2ef3655a-973d-4d69-9b41-21fa4041eed7`
[ingest](https://contribute.data.humancellatlas.org/projects/detail?uuid=2ef3655a-973d-4d69-9b41-21fa4041eed7&tab=project)
**aim**: update linking, keep existing DCP matrices

##### Example project - MVP contributor matrices + DCP generated matrices
short name: pbmcCov19Flu
uuid: `95f07e6e-6a73-4e1b-a880-c83996b3aa5c`
[ingest](https://contribute.data.humancellatlas.org/projects/detail?uuid=95f07e6e-6a73-4e1b-a880-c83996b3aa5c&tab=project) 
**aim**:  update linking, remove MVP matrices because they have been added to the project as normal analysis files, keep existing DCP matrices

### Acceptance criteria for the task
- [ ] update example projects
- [ ] record process and key information as documentation

**Comment by idazucchi** [comment]
next step: nate will prepare a spreadsheet for each project that contains the link uuids and their content
this should allow us to indentify the link files for the DCP generated matrices and MVP matrices, and then delete or keep them according to the need of the project

**Comment by idazucchi** [comment]
we should also discuss whether we want to keep DCP matrices @gabsie to raise this with Tony

**Comment by idazucchi** [comment]
waiting for R35 to be up for review to check the projects look ok in the browser

**Comment by idazucchi** [comment]
`pbmcCov19Flu` --> fixed

**Comment by idazucchi** [comment]
We need to update HumanAdultKidneyLiaoMo project title to match the tracker but because of the issues above the update was stalled 
Summary of the issue : removed fastq files used to produce DCP matrices, replaced with newly converted fastqs from bam files, now DCP matrices are missing their input and causing an [azul error](https://github.com/DataBiosphere/azul/issues/6431)

I started a slack discussion on how to fix HumanAdultKidneyLiaoMo --> thread [here](https://embl-ebi-ait.slack.com/archives/C9XD6L0AD/p1723832977541099)
TL;DR : Hannes suggested to remove the references to the input files of the DCP matrices, rather than the matrices themselves, Nate looked into it and agreed so we are trying this solution for R43

I filled in the import form and will update on progress

**Comment by idazucchi** [comment]
no inexing errors in R43! we can consider this task concluded
